pkg/envtest/server.go

```diff
@@ -82,7 +82,8 @@ var DefaultKubeAPIServerFlags = []string{
 	"--insecure-port={{ if .URL }}{{ .URL.Port }}{{ end }}",
 	"--insecure-bind-address={{ if .URL }}{{ .URL.Hostname }}{{ end }}",
 	"--secure-port={{ if .SecurePort }}{{ .SecurePort }}{{ end }}",
-	"--admission-control=AlwaysAdmit",
+	"--enable-admission-plugins=ValidatingAdmissionWebhook",
+	"--disable-admission-plugins=ServiceAccount",
```

Comment:
This is because of https://github.com/kubernetes-sigs/controller-runtime/pull/787#issuecomment-581039898